### PR TITLE
[Filebeat] Replace copy_from with templated value

### PR DESCRIPTION
**Pull request by andrewkroh**
## What does this PR do?

To ensure compatibility with Elasticsearch versions <7.13 this removes usage of `copy_from` in `set` processors.

## Why is it important?

This will make some modules work with earlier Elasticsearch versions.

## Checklist

- [x] My code follows the style guidelines of this project
~- [ ] I have made corresponding change to the default configuration files~
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Related issues

- Relates #26629

### CODE REVIEW

**Comment by andrewkroh** [comment]
@adriansr I opened this to manually remove `copy_from`. Would you want to help me with this by pushing a change for the two you mention in https://github.com/elastic/beats/pull/26593#issuecomment-871527237.

**Comment by andrewkroh** [comment]
We should probably put a check in place to catch the introduction of `copy_from` in the future. I'm just not sure where that should go.

**Comment by adriansr** [comment]
Pushed those two changes, but my commits are not showing up in this PR?

**Comment by andrewkroh** [comment]
Thanks @adriansr. LGTM

**Review by adriansr — APPROVED** [review]

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -280,6 +280,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Fix bug in `httpjson` that prevented `first_event` getting updated. {pull}26407[26407]
 - Fix bug in the Syslog input that misparsed rfc5424 days starting with 0. {pull}26419[26419]
 - Do not close filestream harvester if an unexpected error is returned when close.on_state_change.* is enabled. {pull}26411[26411]
+- Fix Elasticsearch compatability for modules that use `copy_from` in `set` processors. {issue}26629[26629]
```
Comment (adriansr) [inline CHANGELOG.next.asciidoc:283]:
```suggestion
- Fix Elasticsearch compatibility for modules that use `copy_from` in `set` processors. {issue}26629[26629]
```

**Comment by legoguy1000** [comment]
Instead of actually updating the pipelines themselves, why not add an additional function to automatically change the processor if connecting to an older version of ES? What was just merged in #26676

**Comment by andrewkroh** [comment]
@legoguy1000 We considered that approach in https://github.com/elastic/beats/pull/26593, but during review we identified some edge cases that we couldn't handle by automatic rewriting.

**Comment by andrewkroh** [comment]
run tests

**Comment by legoguy1000** [comment]
> @legoguy1000 We considered that approach in #26593, but during review we identified some edge cases that we couldn't handle by automatic rewriting.

Does the templated value not work with objects?

**Comment by andrewkroh** [comment]
It will run, but the behavior is not the same as with `copy_from`. You end up with a string instead of an object.

**Comment by andrewkroh** [comment]
@Mergifyio backport 7.x
